packages/odata-service-writer/templates/extend/manifest.json

```diff
@@ -10,13 +10,12 @@
                         <% if (annotations.length) { %>
                             <% annotations.forEach(function(annotation, index) { %>
                                 <% if (annotation.technicalName) { %>
-                                    "<%= annotation.name %>"<% if (index < annotations.length - 1) { %>,<% } %>
+                                    "<%= annotation.name %>"<% if (index < annotations.length - 1 || locals.localAnnotationsName) { %>,<% } %>
```

Comment:
`localAnnotationsName` was fine for app generation initially. 
But for update cases it could be limiting as it is possible to support multiple local annotation files in the manifest. In fact the Annotation File manage provides the ability to create them.
Could you address that?

Reply:
`localAnnotationsName` type should be changed then from `string` to `string | string[]` for backward compatibility.